src/components/chart/_macro.njk

```diff
@@ -86,6 +93,17 @@
                     </p>
                 {% endif %}
             {% endif %}
+
+            {% if params.fallbackImageUrl %}
+                <noscript data-fallback-image>
```

Comment:
Rather than use data attributes to identify elements we've tended to use classes

Reply:
I've used an id for this one like we did for the footnotes

Reply:
Are you happy with an id here or would you prefer a class?

Reply:
Id is ok but class would be better just because thats what we've used elsewhere